CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+## How to contribute
+
+The purpose of this page is to provide you with the minimum valuable information about how to contribute to the
+project.
+
+### How reach us 🗞
+
+- you can join the slack community [here](https://androiddevs.it/)
+- and follow the open [discussion on the repository](https://github.com/androiddevelopersitalia/aditogether/discussions)
+
+### Set up the project ⚙️
+
+- Have a JDK >= 11 installed (even the bundled one with AS/IJ)
+- Clone the repository
+- Run `./scripts/tuner.sh` to set up the local tools used by this repository
+
```

Comment:
I would add this bit as well.
Not sure about the wording; as now, I'm referring to Detekt plugin for code-style and Kotest plugin which enables to run single (or set/suite of) Kotest test from IDE and have a nicely formatted and navigable output 


```suggestion
- Make sure to install the required IDEA plugins, from the pop-up shown when the project is opened, to have a smoother experience in the IDE

```